src/pages/reference/apps/get-testnet-zeta.mdx

```diff
@@ -48,6 +48,16 @@ use the built-in `send-zeta` command.
 If you need gas assets on connected chains (like Bitcoin, Sepolia, Mumbai or BSC
 testnet), you can use the other faucets listed below.
 
+### Using the Google Cloud Web3 Faucet
+
+https://cloud.google.com/application/web3/faucet
```

Comment:
This URL takes them straight to the ZetaChain Facuet
https://cloud.google.com/application/web3/faucet/zetachain/testnet

Reply:
https://github.com/zeta-chain/docs/pull/539/commits/e7cb8454b3d5bd24ac5ce20ec913af1e97ebcea5